blazor/predefined-dialogs/position.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Positioning in Blazor Predefined Dialog Component | Syncfusion
+description: Checkout and learn here all about positioning in Syncfusion Predefined Dialogs component and much more details.
```

Comment:
Description: Check out and learn here all about positioning in the Syncfusion Predefined Dialogs component and much more details.